src/clojure_tictactoe/tictactoe.clj

```diff
@@ -52,10 +59,10 @@ Examples:
 
 (defn get-player-choice
   ([]
-   (println "Which space would you like to mark, Player x?")
+   (println (str "Which space would you like to mark, Player " @player-atom "?"))
    (let [choice (parse-move-input (get-user-input))]
    (if choice
-     {choice "x"}
+     {choice @player-atom}
```

Comment:
Do you *need* to use an atom here? I think there's an alternative to using one but could be wrong.

Atom's are *normally* frowned upon in Clojureland. There's usually a better alternative to using them. But in some cases it makes sense.